builtin/mainmenu/dlg_contentstore.lua

```diff
@@ -578,11 +580,18 @@ function store.load()
 	store.aliases = {}
 
 	for _, package in pairs(store.packages_full) do
+		package.url = base_url .. "/packages/" ..
+			package.author:lower():encode_uri_component() ..
```

Comment:
author usernames in URLs should be considered case sensitive, so don't lower them

Reply:
I just preserved the `:lower()` from before, but I can remove it if you wish.

Reply:
The `:lower()` is only on `.id` fields, which is correct

This `.url` isn't used at all it looks like